.github/workflows/reline.yml

```diff
@@ -55,6 +55,50 @@ jobs:
           TERM: xterm-256color
         run: bundle exec rake ci-test
 
+  irb:
+    name: >-
+      irb ${{ matrix.ruby }}  ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        ruby: [ head ]
```

Comment:
Perhaps Ruby 3.2 as well? It'll help us understand if the problem comes from a recent Ruby change.